### PR TITLE
Use GH Actions for container pushing and implement pushing to GHCR

### PR DESCRIPTION
**Pull request by ks129**
Note: Some member from organization have to set username (`GHCR_USER`) and personal access token (`GHCR_PAT`) as secrets to get pushing to GHCR working.

### CODE REVIEW

File: .github/workflows/root.yml [hunk 1]
```diff
@@ -71,14 +71,35 @@ jobs:
           name: Production JAR
           path: build/artifacts/KDBot.jar
 
-      - name: Docker Login
-        run: "docker login -u ${{ secrets.MAVEN_USER }} -p ${{ secrets.MAVEN_PASSWORD }} docker.kotlindiscord.com"
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
```
Comment (gdude2002) [inline .github/workflows/root.yml]: What's this for?

Comment (ks129) [reply .github/workflows/root.yml]: I followed build and push action's official docs, and all examples had it.

Comment (Akarys42) [reply .github/workflows/root.yml]: I'm guessing you should use it if you are targeting something else than AMD64, so we shouldn't need it

Comment (ks129) [reply .github/workflows/root.yml]: Removed in 67d81d3

**Review by gdude2002 — APPROVED** [review]

**Comment by gdude2002** [comment]
This looks fine - I'll update the deployment scripts when @Akarys42 has reviewed/merged.

**Review by Akarys42 — APPROVED** [review]
Looks good to me

**Pull request merged**